log_injection/kemomi_RaspberryPi-xiaozhi/3.py

```
@@ -0,0 +1,505 @@
+            "ip": "192.168.124.38",
+            "mac": "cc:ba:97:20:b4:bc"
+        }
+    }
+
+    try:
+        response = requests.post(OTA_VERSION_URL, headers=header, data=json.dumps(post_data), timeout=10)
+        response.raise_for_status()
+        mqtt_info = response.json()['mqtt']
+        GUI_UPDATE_QUEUE.put(("配置更新成功", "system"))
+        logging.info("配置更新成功")
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)